code/modules/mob/living/carbon/human/life.dm

```diff
@@ -722,7 +722,10 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 	proc/handle_chemicals_in_body()
 
-		if(reagents) //Synths don't process reagents. // fuck you they do now - Iamgoofball
+		if(reagents)
+			var/datum/reagent/R = reagents

```

Comment:
wait what

you're setting the reagents holder as a reagent

this is going to runtime

this needs to be:
for(var/datum/reagent/R in reagents.reagent_list)
if(!R.affected & SYNTHETIC && species.flags & IS_SYNTHETIC) return